pkg/subctl/cmd/join.go

```diff
@@ -77,6 +79,9 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&submarinerDebug, "subm-debug", false, "Enable Submariner debugging (verbose logging)")
 	cmd.Flags().BoolVar(&noLabel, "no-label", false, "skip gateway labeling")
 	cmd.Flags().StringVar(&brokerClusterContext, "broker-cluster-context", "", "Broker cluster context")
+	cmd.Flags().StringVar(&cableType, "cable-type", "ipsec", "Cable type: ipsec or wireguard")
```

Comment:
On an initial merge I would not talk of wireguard, until we finally merge it and have support, otherwise it could look like we already support wireguard, while we not. (We could accept it for development purposes and making our devel lifes easier, but not announce it :-) )

Reply:
+1

Reply:
Done